README.md

```diff
@@ -1,2 +1,16 @@
-# terminology
-Codesystems, questionnaires, and validation tools for them, for the All of Us project.
+# All of Us Terminology
+
+Codesystems, questionnaires, and validation tools for them, for the All of Us
+project.
+
+## Usage
+
+![workflow diagram](doc/overview.png)
+
+1.  Export questionnaires from Vibrent's authoring tool and place them in
+    `source/`.
+1.  `tools/validate_and_publish [--commit]` writes to `generated/`.
```

Comment:
I was initially thinking we publish from what's in the repo. That's how RDR behaves; we check out a tag, complain if there are uncommitted changes [I think/hope], and push from local files.

I'm also glad to say pushing is a required step before deployment. I think I like making it separate from `--commit`.